build.gradle

```diff
@@ -12,7 +12,8 @@ repositories {
 dependencies {
   compile 'com.squareup:javapoet:1.1.0'
   compile 'com.google.auto.service:auto-service:1.0-rc2'
-  compile 'com.google.auto.value:auto-value:1.2-rc1'
+  compile 'com.google.auto.value:auto-value:1.2'
+  compile 'com.gabrielittner.auto.value:auto-value-extension-util:0.1.0'
```

Comment:
Ultimately we decided that since the fix was so trivial that taking on the extra dependency wasn't worth the potential problems in API compatibility that it might otherwise cause. I would rather us target our efforts towards lobbying upstream for a better API to reduce these types of programmer errors.